usaspending_api/database_scripts/matview_generator/universal_transaction_matview.json

```diff
@@ -103,266 +91,192 @@
-    },{
+    }, {
       "name": "type",
       "where": "\"type\" IS NOT NULL",
       "columns": [{"name": "\"type\""}]
-    },{
+    }, {
       "name": "ordered_type",
       "columns": [{"name": "\"type\"", "order": "DESC NULLS LAST"}]
-    },{
+    }, {
       "name": "action_type",
       "columns": [{"name": "\"action_type\""}]
-    },{
+    }, {
       "name": "award_id",
       "columns": [{"name": "\"award_id\""}]
-    },{
+    }, {
       "name": "award_category",
       "where": "\"award_category\" IS NOT NULL",
       "columns": [{"name": "\"award_category\""}]
-    },{
+    }, {
       "name": "total_obligation",
       "where": "\"total_obligation\" IS NOT NULL",
       "columns": [{"name": "\"total_obligation\""}]
-    },{
+    }, {
       "name": "ordered_total_obligation",
       "columns": [{"name": "\"total_obligation\"", "order": "DESC NULLS LAST"}]
-    },{
+    }, {
       "name": "total_obl_bin",
       "columns": [{"name": "\"total_obl_bin\""}]
-    },{
-      "name": "fain",
-      "where": "\"fain\" IS NOT NULL",
-      "columns": [{"name": "\"fain\""}]
-    },{
-      "name": "ordered_fain",
-      "columns": [{"name": "UPPER(\"fain\")", "order": "DESC NULLS LAST"}]
-    },{
-      "name": "gin_fain",
-      "method": "GIN",
-      "columns": [{"name": "UPPER(\"fain\")", "opclass": "gin_trgm_ops"}]
-    },{
-      "name": "piid",
-      "where": "\"piid\" IS NOT NULL",
-      "columns": [{"name": "\"piid\""}]
-    },{
-      "name": "ordered_piid",
-      "columns": [{"name": "UPPER(\"piid\")", "order": "DESC NULLS LAST"}]
-    },{
-      "name": "gin_pid",
-      "method": "GIN",
-      "columns": [{"name": "UPPER(\"piid\")", "opclass": "gin_trgm_ops"}]
-    },{
-      "name": "uri",
-      "where": "\"uri\" IS NOT NULL",
-      "columns": [{"name": "\"uri\""}]
-    },{
-      "name": "ordered_uri",
-      "columns": [{"name": "UPPER(\"uri\")", "order": "DESC NULLS LAST"}]
-    },{
-      "name": "gin_uri",
-      "method": "GIN",
-      "columns": [{"name": "UPPER(\"uri\")", "opclass": "gin_trgm_ops"}]
-    },{
+    }, {
       "name": "pop_country_code",
       "where": "\"pop_country_code\" IS NOT NULL",
       "columns": [{"name": "\"pop_country_code\""}]
-    },{
+    }, {
       "name": "pop_state_code",
       "where": "\"pop_state_code\" IS NOT NULL",
       "columns": [{"name": "\"pop_state_code\""}]
-    },{
+    }, {
       "name": "pop_county_code",
       "where": "\"pop_county_code\" IS NOT NULL",
       "columns": [{"name": "\"pop_county_code\""}]
-    },{
+    }, {
       "name": "pop_zip5",
       "where": "\"pop_zip5\" IS NOT NULL",
       "columns": [{"name": "\"pop_zip5\""}]
-    },{
+    }, {
       "name": "pop_congressional_code",
       "where": "\"pop_congressional_code\" IS NOT NULL",
       "columns": [{"name": "\"pop_congressional_code\""}]
-    },{
-      "name": "latest",
-      "columns": [{"name": "\"latest_transaction_id\""}]
```

Comment:
Don't see the corresponding `latest_transaction_id` being removed from a model?

Reply:
The column was removed in a previous PR. This just removes the index